packages/manager/src/features/Linodes/LinodesLanding/utils.test.ts

```diff
@@ -63,7 +64,7 @@ describe('Linode Landing Utilites', () => {
         interfaces: [...vpcInterfaceList, vpcInterface],
       });
       const vpcIds = getVPCsFromLinodeConfigs([config]);
-      expect(vpcIds).toEqual([2, 3]);
+      expect(vpcIds).toEqual([3, 4, 2]);
```

Comment:
Looks like we've strayed from the goal of the original test here, which introduced a duplicate id and then tested that it's filtered out correctly.

We'd want to keep the expect statement and change the [`vpc_id` on L61 ](https://github.com/linode/manager/pull/10619/files#diff-236647a235dccc8928b844032e447f3936b9038c2426acfd99060dd90ff5aa10R61) to `3` or `4`.

```suggestion
      expect(vpcIds).toEqual([3, 4]);
```

Reply:
yup! ty for catching, addressed